app/code/core/Mage/Adminhtml/Block/Catalog/Product/Edit/Tab/Alerts.php

```diff
@@ -44,15 +44,15 @@ protected function _prepareLayout()
 
         if ($alertPriceAllow) {
             $accordion->addItem('price', [
-                'title'     => Mage::helper('adminhtml')->__('Price alert subscription was saved.'),
-                'content'   => $this->getLayout()->createBlock('adminhtml/catalog_product_edit_tab_alerts_price')->toHtml() . '<br />',
+                'title'     => Mage::helper('adminhtml')->__('Product price alert'),
+                'content'   => $this->getLayout()->createBlock('adminhtml/catalog_product_edit_tab_alerts_price')->toHtml(),
                 'open'      => true
             ]);
         }
         if ($alertStockAllow) {
             $accordion->addItem('stock', [
-                'title'     => Mage::helper('adminhtml')->__('Stock notification was saved.'),
-                'content'   => $this->getLayout()->createBlock('adminhtml/catalog_product_edit_tab_alerts_stock'),
+                'title'     => Mage::helper('adminhtml')->__('Product stock alert'),
```

Comment:
1. This should be changed to "Back in Stock Alerts". Using Product as prefix is like an abuse in my opinion. Also I keep the correct description from configuration
2. It needs a row for translation in Mage_Catalog.csv